generators/php/codegen/src/ast/Field.ts

```diff
@@ -53,7 +59,7 @@ export class Field extends AstNode {
         }
 
         this.type.write(writer);
-        writer.write(` ${this.name}`);
+        writer.write(` ${this.name.startsWith("$") ? "" : "$"}${this.name}`);
```

Comment:
nit: It might be worth making a helper function for this.

Reply:
i think we'll wanna remove this eventually so leaving as is for now